database_schemas/cassandra/benchflow.cql

```diff
@@ -210,128 +209,175 @@ CREATE TABLE exp_process_duration (
+  throughput_me double,
+  throughput_median double,
+  throughput_mode list<double>,
+  throughput_sd double,
   experiment_id text,
   PRIMARY KEY ((experiment_id))
 );
 
 CREATE TABLE exp_number_of_process_instances (
-  number_of_process_instances_avg float,
-  number_of_process_instances_min float,
-  number_of_process_instances_max float,
-  number_of_process_instances_q1 float,
-  number_of_process_instances_q2 float,
-  number_of_process_instances_q3 float,
-  number_of_process_instances_p95 float,
-  number_of_process_instances_ci095_min float,
-  number_of_process_instances_ci095_max float,
-  number_of_process_instances_me float,
-  number_of_process_instances_median float,
-  number_of_process_instances_mode list<float>,
-  number_of_process_instances_sd float,
+  number_of_process_instances_avg double,
+  number_of_process_instances_min bigint,
+  number_of_process_instances_max bigint,
+  number_of_process_instances_q1 bigint,
+  number_of_process_instances_q2 bigint,
+  number_of_process_instances_q3 bigint,
+  number_of_process_instances_p95 bigint,
+  number_of_process_instances_ci095_min double,
+  number_of_process_instances_ci095_max double,
+  number_of_process_instances_me double,
+  number_of_process_instances_median bigint,
+  number_of_process_instances_mode list<bigint>,
+  number_of_process_instances_sd double,
   experiment_id text,
   PRIMARY KEY ((experiment_id))
 );
 
 CREATE TABLE exp_byte_size (
-  size_avg float,
-  size_min float,
-  size_max float,
-  size_q1 float,
-  size_q2 float,
-  size_q3 float,
-  size_p95 float,
-  size_ci095_min float,
-  size_ci095_max float,
-  size_me float,
-  size_median float,
-  size_mode list<float>,
-  size_sd float,
+  size_avg double,
+  size_min bigint,
+  size_max bigint,
+  size_q1 bigint,
+  size_q2 bigint,
+  size_q3 bigint,
+  size_p95 bigint,
+  size_ci095_min double,
+  size_ci095_max double,
+  size_me double,
+  size_median bigint,
+  size_mode list<bigint>,
+  size_sd double,
   experiment_id text,
   PRIMARY KEY ((experiment_id))
 );
 
+CREATE TABLE exp_io (
+  reads_avg double,
+  reads_min bigint,
+  reads_max bigint,
+  reads_q1 bigint,
+  reads_q2 bigint,
+  reads_q3 bigint,
+  reads_p95 bigint,
+  reads_ci095_min double,
+  reads_ci095_max double,
+  reads_me double,
+  reads_median bigint,
+  reads_mode list<bigint>,
+  reads_sd double,
+  writes_avg double,
+  writes_min bigint,
+  writes_max bigint,
+  writes_q1 bigint,
+  writes_q2 bigint,
+  writes_q3 bigint,
+  writes_p95 bigint,
+  writes_ci095_min double,
+  writes_ci095_max double,
+  writes_me double,
+  writes_median bigint,
+  writes_mode list<bigint>,
+  writes_sd double,
+  total_avg double,
+  total_min bigint,
+  total_max bigint,
+  total_q1 bigint,
+  total_q2 bigint,
+  total_q3 bigint,
+  total_p95 bigint,
+  total_ci095_min double,
+  total_ci095_max double,
+  total_me double,
+  total_median bigint,
+  total_mode list<bigint>,
+  total_sd double,
+  experiment_id text,
+  device text,
```

Comment:
the io_data contains many rows, with io_data_id identifying single rows. If anything we need the container_id, since these data are for a certain container